### PR TITLE
added new workload for catalog item cee-cf-110

### PR DESCRIPTION
**Pull request by ajblum**
##### SUMMARY

added a new task that injects grading script for cee-cf-110 into bastion host

##### ISSUE TYPE

- New role Pull Request

##### COMPONENT NAME

cee-cf-110
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: ansible/roles_ocp_workloads/ocp4_workload_cee-cf-110/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# your workload.
+
+# Documentation for variable 1 (what does it do?)
+#ocp4_workload_example_variable_1: "value 1"
+
+# Documentation for variable 2 (what does it do?)
+#ocp4_workload_example_variable_2: "value 2"
+
+# PnTAE course id should match the name of the grading script, catalog item, and LMS course_code
+ocp4_workload_example_pntae_course_id: "cee-cf-110"
```
Comment (wkulhanek) [inline ansible/roles_ocp_workloads/ocp4_workload_cee-cf-110/defaults/main.yml:21]: That can't be quite right, can it? Shouldn't it be `ocp4_workload_cee_cf_110_pntae_course_id` ?

File: ansible/roles_ocp_workloads/ocp4_workload_cee-cf-110/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  role_name: ocp4_workload_example
+  author: Red Hat GPTE, Wolfgang Kulhanek (wkulhane@redhat.com)
```
Comment (wkulhanek) [inline ansible/roles_ocp_workloads/ocp4_workload_cee-cf-110/meta/main.yml:4]: Nope. I didn't write this. :-) Can't support it either... :-) role_name and description need updating, too, please.

File: ansible/roles_ocp_workloads/ocp4_workload_cee-cf-110/readme.adoc [hunk 3]
```diff
@@ -0,0 +1,188 @@
+= ocp4_workload_example - Example Workload Role
```
Comment (wkulhanek) [inline ansible/roles_ocp_workloads/ocp4_workload_cee-cf-110/readme.adoc:1]: Please update with role name. And description.

File: ansible/roles_ocp_workloads/ocp4_workload_cee-cf-110/tasks/remove_workload.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+# Implement your workload removal tasks here
+# ------------------------------------------
+
+
```
Comment (wkulhanek) [inline ansible/roles_ocp_workloads/ocp4_workload_cee-cf-110/tasks/remove_workload.yml:5]: No removal logic?

File: ansible/roles_ocp_workloads/ocp4_workload_cee-cf-110/meta/main.yml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
```
Comment (stencell) [inline ansible/roles_ocp_workloads/ocp4_workload_cee-cf-110/meta/main.yml:2]: Please make sure these are all updated

File: ansible/roles_ocp_workloads/ocp4_workload_pntae/defaults/main.yml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+# your workload.
+
+# Documentation for variable 1 (what does it do?)
+#ocp4_workload_example_variable_1: "value 1"
+
+# Documentation for variable 2 (what does it do?)
+#ocp4_workload_example_variable_2: "value 2"
+
+# PnTAE course id should match the name of the grading script, catalog item, and LMS course_code
+ocp4_workload_example_pntae_course_id: "cee-cf-110"
```
Comment (wkulhanek) [inline ansible/roles_ocp_workloads/ocp4_workload_pntae/defaults/main.yml:21]: Still example in the variable name.

File: ansible/roles_ocp_workloads/ocp4_workload_pntae/tasks/workload.yml [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+# To Do: Implement your workload deployment tasks here
+# -------------------------------------------------------------------------
+
+- name: Copy lab grading script onto bastion hosts
+  copy:
+    src: "../files/{{ ocp4_workload_example_pntae_course_id }}"
+    dest: "/usr/local/bin/{{ ocp4_workload_example_pntae_course_id }}"
```
Comment (wkulhanek) [inline ansible/roles_ocp_workloads/ocp4_workload_pntae/tasks/workload.yml:12]: Also ....example...

File: ansible/roles_ocp_workloads/ocp4_workload_pntae/tasks/remove_workload.yml [hunk 8]
```diff
@@ -0,0 +1,19 @@
+---
+# Implement your workload removal tasks here
+# ------------------------------------------
+
+- name: remove file
+  file:
+    path: "/usr/local/bin/{{ ocp4_workload_example_pntae_course_id }}"
```
Comment (wkulhanek) [inline ansible/roles_ocp_workloads/ocp4_workload_pntae/tasks/remove_workload.yml:7]: ....example...

**Comment by wilson-walrus** [comment]
I worked with Andrew to get most of the changes complete. Currently awaiting final push to fix remaining issues.

File: ansible/roles_ocp_workloads/ocp4_workload_pntae/defaults/main.yml [hunk 9]
```diff
@@ -3,19 +3,5 @@ become_override: False
...
-
-# Documentation for variable 1 (what does it do?)
-#ocp4_workload_example_variable_1: "value 1"
-
-# Documentation for variable 2 (what does it do?)
-#ocp4_workload_example_variable_2: "value 2"
-
 # PnTAE course id should match the name of the grading script, catalog item, and LMS course_code
-ocp4_workload_example_pntae_course_id: "cee-cf-110"
+ocp4_workload_example_course_id: "cee-cf-110"
```
Comment (wkulhanek) [inline ansible/roles_ocp_workloads/ocp4_workload_pntae/defaults/main.yml:7]: You got it right in the other two files... this one also has to be `ocp4_workload_pntae_course_id`

**Pull request merged**